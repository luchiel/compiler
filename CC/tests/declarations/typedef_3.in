typedef struct a {int b;} c;
