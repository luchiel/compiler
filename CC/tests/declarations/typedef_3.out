type <int>
type <float>
type <void>
type <0>
	struct 0
		tag a
		variable b of type <int>
type <c>
	alias to type <0>
